openwisp_radius/tests/test_commands.py

```diff
@@ -225,6 +225,27 @@ def _create_old_users():
             )
             self.assertEqual(User.objects.count(), 1)
 
+        with self.subTest('Users which have accounting sessions should not be deleted'):
+            _create_old_users()
+            user = self._create_user(date_joined=now() - timedelta(days=3))
+            RegisteredUser.objects.create(
+                user=user,
+                method='email',
+                is_verified=False,
+            )
+            opts = dict(
+                username=user.username,
+                unique_id='2',
+                nas_ip_address='127.0.0.1',
+                session_id='1',
+            )
```

Comment:
```suggestion
            opts = options = _RADACCT.copy()
            options['username'] = user.username
```
Consider using above since it is how other tests are written.